build.gradle

```diff
@@ -55,7 +55,7 @@ ext {
     rxBindingsVersion = '2.1.1'
 
     //Kotlin
-    kotlinStdlibVersion = '1.2.71'
+    kotlinStdlibVersion = '1.3.11'
```

Comment:
Aren't this the same thing? Can we remove one of the vars?

Reply:
Good question. I guess one is for the `kotlin-gradle-plugin` and the other one is for the `kotlin-stdlib`. I really don't know why we still use different vars once: 

>  Starting with Kotlin 1.1.2, the dependencies with group org.jetbrains.kotlin are by default resolved with the version taken from the applied plugin.

I would say we can remove the `kotlinStdlibVersion` and start using the `kotlinVersion` only. I'll take care of it in this PR.

Reply:
You must also change on the project we use this as submodule to avoid breaking it.